app/Traits/AuthorizeInclude.php

```diff
@@ -45,9 +45,9 @@ public function authorizeInclude(string $class, ?string $requestedInclude = null
             $permission = array_key_exists(
                 $include,
                 $relationPermMap
-            ) ? $relationPermMap[$include] : $this->camelToDashed($include);
+            ) ? $relationPermMap[$include] : 'read-'.$this->camelToDashed($include);
 
-            if (Auth::user()->cant('read-'.$permission)) {
+            if (Auth::user()->cant($permission)) {
                 Log::debug('User is missing permission: read-'.$permission);
```

Comment:
this doesn't seem to account for the new definition of `$permission` above

Reply:
Oop good catch